examples/jupyter/import_config_esgf.ipynb

```diff
@@ -0,0 +1,55 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bf9da680",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pan3d import DatasetBuilder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f052f51-86f1-4b44-bfa9-0c6d75892571",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config_path = '../example_config_esgf.json'\n",
+    "builder = DatasetBuilder(esgf=True)\n",
+    "\n",
+    "# optional: to apply UI config, access viewer before importing\n",
+    "viewer = builder.viewer\n",
+    "\n",
+    "builder.import_config(config_path)\n",
```

Comment:
Thinking out load, but maybe the import_config could take a kwarg to force viewer creation like when needed.

`builder.import_config(config_path, ui=True)`

Reply:
I think that including the kwarg on the Builder constructor is sufficient for this. Do you think that we should have both?

Reply:
yes